assets/css/src/newsletter_editor/components/save.styl

```diff
@@ -65,6 +65,9 @@
 .mailpoet_save_next, .mailpoet_save_button_group
   float: right
 
+.mailpoet_save_next
+  margin-left: 10px !important
```

Comment:
I think that 10px is more than the original gap was (https://d.pr/i/w7XWz0).
Also there is no need to use important. I think we should avoid using !important when it is important. You can make selectivity of this rule stronger by adding one more class to its definition e.g. `.mailpoet_save_wrapper .mailpoet_save_next`